assets/test/test_bundle_parse.js

```diff
@@ -84,23 +84,23 @@ describe('parseBundlePlugin', function() {
         it('should have its name set to Kolibri', function (done) {
             var data = {
                 name: "kolibri.plugin.test.test_plugin",
-                entry_file: "src/file.js",
+                src_file: "src/file.js",
                 external: true,
                 core: true,
                 stats_file: "output.json",
-                module_path: "kolibri/plugin/test"
+                static_dir: "kolibri/plugin/test"
             };
             assert.equal(parseBundlePlugin(data, "/")[0].output.library, "Kolibri");
             done();
         });
     });
 });
 
-describe('readBundlePlugins', function() {
+describe('readBundlePluginss', function() {
```

Comment:
garh, double up on search and replace... I pluralized that plugin to emphasize that it's reading multiple bundles.